### PR TITLE
fix package.sh test to see if Xvfb is running

### PR DESCRIPTION
**Pull request by mogoodrich**
The test to see if Xvfb is running does not appear to be working correctly.  See my inline comments.

### CODE REVIEW

File: ui/scripts/package.sh [hunk 1]
```diff
@@ -13,8 +13,7 @@ rm -rf $ROOT_DIR/target/${ZIP_FILE_NAME}*.zip
 npm install
 bower install
 
-pids=$(pgrep Xvfb)
```
Comment (mogoodrich) [inline ui/scripts/package.sh]: If Xvfb is not running, the exit code of the pgrep will be 1, and therefore the exit code of the variable assignment is also 1. Since "set -e" is set earlier in the commit, the script will always exit at this point.

File: ui/scripts/package.sh [hunk 2]
```diff
@@ -13,8 +13,7 @@ rm -rf $ROOT_DIR/target/${ZIP_FILE_NAME}*.zip
...
-if [ -n "$pids" ]; then
+if [ -z $(pgrep Xvfb) ]; then
```
Comment (mogoodrich) [inline ui/scripts/package.sh]: Putting the pgrep directly within the test prevents a non-zero exit code from prematurely terminating the script.

Also note that I believe we should be testing with -z here, not -n, since we want to return true if the value DOES have zero size.

**Comment by mogoodrich** [comment]
Have updated, will issue new pull request, closing this one out.
